USER-INTERFACE-INTERFACE.md

```diff
@@ -560,18 +561,31 @@ of payments that have not yet been confirmed. Only cumulative account balances a
 ```
 "payload": {
     "dbPassword": <string>,
-    "mnemonicPhraseSize": <number>,
-    "mnemonicPhraseLanguage": <string>,
-    "mnemonicPassphraseOpt": <optional string>,
-    "consumingDerivationPath": <string>,
-    "earningDerivationPath": <string>
+    "seedSpecOpt": {
+        "mnemonicPhraseSize": <number>,
+        "mnemonicPhraseLanguage": <string>,
+        "mnemonicPassphraseOpt": <optional string>
+    },
+    "consumingDerivationPathOpt": <optional string>,
+    "earningDerivationPathOpt": <optional string>
 }
 ```
 ##### Description:
-This message directs the Node to generate a pair of wallets and report their mnemonic phrase and their addresses
-back to the UI. If the database already contains a wallet pair, the wallet generation will fail.
+This message directs the Node to generate a pair of wallets and report their vital statistic back to the UI. 
+If the database already contains a wallet pair, the wallet generation will fail.
+
+Wallets can be generated in two ways:
+
+* Using derivation paths from a seed: in this case the request should contain a specification for the seed and 
+  a derivation path for each wallet, and the response will contain a mnemonic phrase representing the generated
+  seed, along with the address and private key for each wallet generated.
+* Entirely at random: in this case no direction should be given in the request, and the response will contain
+  only the address and private key for each wallet generated.
 
-`dbPassword` is the current database password. If this is incorrect, the wallet generation will fail.
+`dbPassword` is the current database password. If this is incorrect or absent, the wallet generation will fail.
+
+`seedSpecOpt` gives the parameters for generating the seed. This only makes sense if one or more of the 
+derivation paths is supplied. If no derivation paths are supplied, this parameter is ignored.
 
 `mnemonicPhraseSize` is the number of words that should be generated in the mnemonic phrase. The acceptable values
```

Comment:
I believe we have changed the default phrase size to 12 words, so may need to adjust the wording here.

@FinsaasGH @bertllll I'm pretty confident this was changed - can you confirm?

Reply:
This was changed to default to 12 words

Reply:
No, the default here is 24 words. The GUI project decided that if their user didn't specify a phrase length, they would specify 12 words here, but the default for Node if you don't specify this value is still 24 words, as it should be. Node defaults to more security; the GUI folks default to less, for user experience.

Reply:
that makes sense so thanks for clarifying @dnwiebe